pkg/tasks/metricsserver.go

```diff
@@ -40,6 +40,32 @@ func (t *MetricsServerTask) Run(ctx context.Context) error {
 }
 
 func (t *MetricsServerTask) create(ctx context.Context) error {
+	{
+		// TODO: This is a temporary workaround until the requirements for https://github.com/openshift/cluster-monitoring-operator/pull/2329
```

Comment:
put it here to make it easier to revert + to not bend `MetricsClientCATask` and make it look as a good fix.